src/measure_theory/card_measurable_space.lean

```diff
@@ -30,52 +30,52 @@ variables {α : Type u}
 open_locale cardinal
 open cardinal set
 
-local notation `ω₁` := (aleph 1 : cardinal.{u}).ord.out.α
+local notation `ℵ₀₁` := (aleph 1 : cardinal.{u}).ord.out.α
```

Comment:
Should this be `ℵ₁`?
```suggestion
local notation `ℵ₁` := (aleph 1 : cardinal.{u}).ord.out.α
```

Reply:
Yeah, that was a find + replace accident.

Reply:
It's used as an ordinal (e.g. `generate_measurable_rec` is defined by recursion using the well-order on `ω₁`), so I'd argue `ω₁` is used aptly here. There are several ℵ₀₁ below that needs to be replaced back.